packages/fluentui/react-bindings/src/hooks/useFrameFocusDispatch.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+export const FUI_FRAME_EVENT = 'fuiframefocus';
+
+export const useFrameFocusDispatch = (
+  enableFrameFocusDispatch: boolean,
+  $document: Document,
+  intervalMS: number = 1000,
```

Comment:
nit let's have a more meaningful name
```suggestion
  pollDuration: number = 1000,
```